src/helix-ui/elements/HXFileTileElement.js

```diff
@@ -154,7 +174,7 @@ export class HXFileTileElement extends HXElement {
      * @type {String}
      */
     get name () {
-        return this.getAttribute('name' || '');
+        return this.getAttribute('name') || '';
```

Comment:
Seems we need to do some rebasing of branches. This was merged to `master`, but doesn't seem to be present in `css-audits` or any of its child branches.